src/material/datepicker/datepicker.ts

```diff
@@ -210,8 +210,21 @@ export class MatDatepicker<D> implements OnDestroy, CanColor {
    */
   @Output() readonly monthSelected: EventEmitter<D> = new EventEmitter<D>();
 
-  /** Classes to be passed to the date picker panel. Supports the same syntax as `ngClass`. */
-  @Input() panelClass: string | string[];
+  private _panelClass: string[];
+
+  /** Classes to be passed to the date picker panel. */
+  get panelClass(): string | string[] {
+      return this._panelClass;
+  }
+
+  /**
+   * Classes to be passed to the date picker panel.
+   * Supports string and string array values, similar to `ngClass`.
+   */
+  @Input()
+  set panelClass(value: string | string[]) {
+      this._panelClass = coerceStringArray(value);
```

Comment:
It looks like datepicker doesn't have any tests for `panelClass` right now because there wasn't any logic to it before; it would be good to add a couple here now